crawler/emitter.py

```diff
@@ -241,6 +240,61 @@ def _publish_to_stdout(self):
                     print line.strip()
                     sys.stdout.flush()
 
+    def _publish_to_logstash(self, url, max_emit_retries=5):
```

Comment:
That new option should be a --format option. You should use the same http emitter, with different --format options. Let's say this new one is called `json-per-line` or something like that.